s3-cloudfront.config.yaml

```diff
@@ -1 +1,5 @@
-bucket_name: frontend.${EnvironmentName}.${DnsDomain}
\ No newline at end of file
+bucket_name: frontend.${EnvironmentName}.${DnsDomain}
+bucket_policy:
```

Comment:
I think the bucket_policy shouldn't be enabled by default. Just put this block in but comment it out as an example and implement a test for this config. See the https://github.com/theonestack/hl-component-cloudfront/tree/master/tests as an example for creating a test